app/Jobs/CreateArticle.php

```diff
@@ -56,6 +59,10 @@ public function handle(): Article
         $article->authoredBy($this->author);
         $article->syncTags($this->tags);
 
+        if (App::environment() !== 'testing') {
```

Comment:
Could we perhaps create a new `ArticleWasCreated` event and fire this in a listener similar to how we do when [notifying users when articles are approved](https://github.com/laravelio/laravel.io/blob/main/app/Providers/EventServiceProvider.php#L24-L26)?